Problems/2580/ans_2580_JHP(2).py

```diff
@@ -0,0 +1,73 @@
+# https://www.acmicpc.net/problem/2580
+# 문제 : 9x9 스도쿠 문제 풀기
+# 제한 : PyPy3 1172ms 이하
+
+# 접근방법 : DFS(깊이 우선 탐색) 및 백트래킹(브루트 포스) 사용
+# 제한 조건이 있으므로, 유망한 숫자 검사와 재귀를 최소화 해야함.
+
+import sys
+
+def num_check(x,y):    
+    defNum= [1,2,3,4,5,6,7,8,9]
+    
+    # (1) 가로/세로 비교
+    for i in range(9):
+        if board[x][i] in defNum:
+            defNum.remove(board[x][i])         
+        if board[i][y] in defNum:
+            defNum.remove(board[i][y])     
```

Comment:
set을 사용한다면, 이런식으로 처리가 가능
```suggestion
    for i in range(9):
        defNum -= board[x][i]         
        defNum -= board[i][y]     
```